### DISCUSSION TITLE
作者大大是否能将巅峰杯的比赛奖励进度显示在每次玩家比赛后呢？

### DISCUSSION THREAD

**Discussion by EEEEEEEEdison** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#45] [Ideas]
作者大大辛苦，只是感觉这样养起来会更方便一点。

**Comment by EtherealAO** [comment]
比赛奖励进度是什么？

**Reply by EEEEEEEEdison** [reply]
巅峰杯重赏累积有奖励，比如经典三冠，秋三冠，我意思是有没有可能做一个每当玩家完成一场比赛的冠军时（比如玩家赢得了皋月赏），就会提示经典三冠（1/3）+10+10，将该比赛关联的所有奖励目标内容以及奖励属性点列出来供玩家参考。

**Reply by trojan-uma** [reply]
感觉必要性不是很大的说，主要是因为也就那几样玩过一段时间基本上就能背出来了
